projects/packages/external-media/src/class-external-media.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Register the external media to both WP Admin and Editor.
+ *
+ * @package automattic/jetpack-external-media
+ */
+
+namespace Automattic\Jetpack;
+
+/**
+ * Class External_Media
+ */
+class External_Media {
+	const PACKAGE_VERSION = '0.1.0-alpha';
+	const BASE_DIR        = __DIR__ . '/';
+	const BASE_FILE       = __FILE__;
+
+	/**
+	 * Add hooks and filters.
+	 */
+	public static function init() {
+		add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'enqueue_block_editor_assets' ) );
+	}
+
+	/**
+	 * Enqueue block editor assets.
+	 */
+	public static function enqueue_block_editor_assets() {
+		$assets_base_path = 'build/';
+		$asset_name       = 'jetpack-external-media-editor';
+
+		Assets::register_script(
+			$asset_name,
+			$assets_base_path . "$asset_name/$asset_name.js",
+			__FILE__,
+			array(
+				'enqueue'      => true,
+				'textdomain'   => 'jetpack-external-media',
+
+				/**
+				 * It depends on the `jetpack-blocks-editor` since the feature requires `Jetpack_Editor_Initial_State`.
```

Comment:
That seems problematic if you later want to use External Media outside of the block editor.

Would it be worth it to build a new set of variables with just what you'll need for External Media, enqueue that and use it if `Jetpack_Editor_Initial_State` is not available?

Reply:
Hey there, Arthur. Yes, go ahead. We discussed the feature flag here and it will be removed soon, as it is out of beta for some time now, so this check will be temporary.

Reply:
Thanks 👍